### ISSUE TITLE
Is is still working?

### ISSUE THREAD

**Issue by nghichcode** [ghoost82/mijia-720p-hack#12]
Is is still working with mijia firmware 2018?

**Comment by Unclezz** [comment]
Not for me. I am currently running on version 3.3.6_2018061418 and the camera is not getting the firmware from the SDCARD. 
I configured on SDCARD a different WiFi network than the original one and it simply doesn't appear in the new network but in the old one.

There is a way to rollback to previous firmware... just go for it!!

**Comment by nghichcode** [comment]
Solution is you downgrade your firmware! And re install hack!
This seem, this project has been canceled! So i will creat a new project and develope code for firmware 2018 in someday :smile:

**Comment by ioldfish** [comment]
> Solution is you downgrade your firmware! And re install hack!
> This seem, this project has been canceled! So i will creat a new project and develope code for firmware 2018 in someday 😄

it not work well for 2018，did you done ？help

**Comment by fliphess** [comment]
For me, using a firmware from early this year (2018), I managed to get it working @ioldfish

**Comment by gjj** [comment]
Not for the latest firmware. It works for 3.3.6_2017111315 though if you're able to find the firmware.

**Comment by cesarlopes** [comment]
How can I downgrade the firmware?

**Comment by gjj** [comment]
@cesarlopes I think you can find it somewhere in the issues list. Although I can send it to you if you drop me an email.

**Comment by cesarlopes** [comment]
Hi @JiaJian do you have the firmware for the MJSXJ01CM model? If you can send me my e-mail is cesarlopesmx@gmail.com thank you so much.

**Comment by DanielePalombo** [comment]
Is it working with MIXJ04CM and firmware 3.3.6_2018070914? Or anyone can post the firmware for my model?

**Comment by gjj** [comment]
@DanielePalombo it isn't working for 3.3.6_2018070914.

**Comment by DanielePalombo** [comment]
Thanks for the fast response. Do you know where I can find the downgraded firmware for MIXJ04CM ?

**Comment by Undefined82** [comment]
Hi, I downgraded the camera firmware to the old one as per the BT. But still cannot get the hack to work. I cannot see anything in the http://camera_ip/. Is the pre-compiled folder the same from that is downloaded from the ZIP file?

**Comment by Staars** [comment]
> Hi, I downgraded the camera firmware to the old one as per the BT. But still cannot get the hack to work. I cannot see anything in the http://camera_ip/. Is the pre-compiled folder the same from that is downloaded from the ZIP file?

Same here. Downgrade (and telnet) is possible, but the hack does not come to live. I get a movie folder and log output on the SD-card.

**Comment by nmhung1985** [comment]
> @DanielePalombo it isn't working for 3.3.6_2018070914.

Is this fw version also available on MJSXJ01CM? If it is, it looks like MJSXJ01CM is MIXJ04CM.

**Comment by nmhung1985** [comment]
> Solution is you downgrade your firmware! And re install hack!
> This seem, this project has been canceled! So i will creat a new project and develope code for firmware 2018 in someday 😄


Erm, maybe you wanna take a look at the official version distributed in VN? MIXJ04CM

**Comment by kaczmar986** [comment]
> Erm, maybe you wanna take a look at the official version distributed in VN? MIXJ04CM

What do you mean? How can I do that? Do you have some information that I can start my research with?

**Comment by nmhung1985** [comment]
> > Erm, maybe you wanna take a look at the official version distributed in VN? MIXJ04CM
> 
> What do you mean? How can I do that? Do you have some information that I can start my research with?

No, I was just talking to the guy who said he would develop fw for the 2018 version.

**Comment by ighorn1** [comment]
Hi I have three same camera I downgrade one to firmware 3.3.6_2017080313 and 3.3.6_2017071114 and both didn't works I also got a movie folder and log output on the SD-card when I installed the hack. Where is possible to find firmware 3.3.6_2017111315?

**Comment by perinoid** [comment]
It doesn't work for me. But I wonder, where is the **manufacture.bin** file...

**Comment by joaoandredcruz** [comment]
I have 3.3.6_2017080313, and can't make it work. Anyone with same problem?

**Comment by TiNNauR** [comment]
> Where is possible to find firmware 3.3.6_2017111315?

I have same problem. Anybody can help?

**Comment by TiNNauR** [comment]
> @cesarlopes I think you can find it somewhere in the issues list. Although I can send it to you if you drop me an email.

Can you send it to my email?
tinnaur@ukr.net
Thanks
